src/main/scala/com/fulcrumgenomics/vcf/JointVariantContextIterator.scala

```diff
@@ -32,38 +32,28 @@ object JointVariantContextIterator {
   def apply(iters: Seq[Iterator[VariantContext]],
             dict: SequenceDictionary
            ): JointVariantContextIterator = {
-    new JointVariantContextIterator(
-      iters=iters,
-      dictOrComp = Left(dict)
-    )
+    new JointVariantContextIterator(iters = iters, dict = dict)
   }
 
+  @deprecated("VariantContextComparator will no longer compare variant contexts on location alone.")
   def apply(iters: Seq[Iterator[VariantContext]],
```

Comment:
This entrypoint was never used in this codebase. But it is no longer possible to use `VariantContextComparator` since it compares variants beyond location alone. Seems like my only option is deprecation and raising an exception.